daemon/daemon.go

```diff
@@ -389,5 +390,10 @@ func (d *daemon) Run() error {
 	reflection.Register(s)
 	log.Info().Msg("Reflection Registration is called.... ")
 
+	learningMaterialAPI, err := api.NewLearningMaterialAPI()
+	if err == nil {
+		go http.ListenAndServe(":8888", learningMaterialAPI.Handler())
```

Comment:
Can't you add this handler to the existing Haaukins Mux that also serves all events. Why another web server?